packages/react-scripts/config/webpack.config.js

```diff
@@ -55,6 +57,10 @@ const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== 'false';
 // Some apps do not need the benefits of saving a web request, so not inlining the chunk
 // makes for a smoother build process.
 const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
+// We might not want to use the hard source plugin on environments that won't persist the cache for later
+const useHardSourceWebpackPlugin =
```

Comment:
Rather than an env variable this might be better as a config value instead?

Reply:
As it's experimental (mainly because https://github.com/mzgoddard/hard-source-webpack-plugin/pull/419 + waiting for Webpack 5), I've found value in flipping it on/off at will - so was thinking the environment variables _maybe_ gave more flexibility.

If there's cleaner way without needing to make a code change in the consuming project that would still be good.